### PR TITLE
Fixes #2571 - Adds a dashboard route with the list of current dashboards

### LINKED ISSUE #2571: Create a landing page for all the Dashboards we have
**Environment:**
Browser / Version: Chrome 68 (Desktop), Firefox Nightly 63.0a1 (Desktop)
Operating System: Windows 10

**Steps to Reproduce:**
1. Navigate to https://webcompat.com/dashboard/

**Expected Behavior:**
All dashboards, together with http://adamstevenson.ca/mozilla/reports/ and https://webcompat.com/dashboard/triage are listed here.

**Actual Behavior:**
The page is a 404.

**Notes:**
1. Screen capture attached.

Watchers:
@softvision-sergiulogigan
@softvision-oana-arbuzov

sv;

image

### PR DESCRIPTION
**Pull request by karlcow**
This PR fixes issue #2571

## Proposed PR background

As requested by @softvision-sergiulogigan this is a page which collects a list of different dashboard pages, now that we have a couple of them. This might need to be adjusted in the future.

### CODE REVIEW

**Comment by magsout** [comment]
👍

File: webcompat/views.py [hunk 1]
```diff
@@ -439,6 +439,14 @@ def cssfixme():
     return (msg, 410, {'content-type': 'text/plain; charset=utf-8'})
 
 
+@app.route('/dashboard/')
```
[the remark this answers is not in the document]
Comment (karlcow) [reply webcompat/views.py]: Let's modify this and I will merge. Thanks @miketaylr

**Comment by karlcow** [comment]
Thanks @magsout for the review.

File: webcompat/views.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (karlcow) [reply webcompat/views.py]: fixed

**Comment by karlcow** [comment]
Merging after following changes request from @miketaylr and approval by @magsout

**Pull request merged**